app/controllers/organizations_controller.rb

```diff
@@ -0,0 +1,49 @@
+class OrganizationsController < ApplicationController
```

Comment:
I believe we don't need any of this methods. For the CRUD of organizations we use the admin system but we don't need this services